autobotAI_integrations/integrations/microsoft_office_365/__init__.py

```
@@ -164,6 +163,14 @@ def build_python_exec_combinations_hook(
                 "context": payload_task.context,
             }
         ]
+        try:
+            defender_client = msgraph.GraphServiceClient(
+                credentials=credential, scopes=scopes + ["https://api.security.microsoft.com/.default"]
```

Ruff (0.15.13)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)